### PR TITLE
Send onboarding email

### PR DESCRIPTION
**Pull request by zrthxn**
Starting onboarding email

### CODE REVIEW

**Comment by zrthxn** [comment]
@abbasidaniyal Check this out and tell me if this is the right direction

File: subscribers/tasks.py [hunk 1]
```diff
@@ -7,6 +7,10 @@
 from subscribers.utils import check_and_notify_subscriber
 
 
+def welcome_user():
```
Comment (abbasidaniyal) [inline subscribers/tasks.py:10]: This is not needed. This is used for periodic tasks generally.

File: subscribers/utils.py [hunk 2]
```diff
@@ -16,6 +16,19 @@
 twilio_client = Client(settings.TWILIO_ACCOUND_SID, settings.TWILIO_AUTH_TOKEN)
 
 
+def send_onboarding_mail(subscriber_email):
```
Comment (abbasidaniyal) [inline subscribers/utils.py:19]: Yep this is good. Just add the decorator to make it a celery task.

**Review by abbasidaniyal — CHANGES_REQUESTED** [review]
Yep this looks good. You  need to call the send welcome_user() from views.py rest everything LGTM

**Pull request merged**